lang/ilias_fr.lang

```diff
@@ -2727,7 +2726,7 @@ chatroom_adm#:#client_settings#:#Paramètres Client
 chatroom_adm#:#enable_osd#:#Activer les Notifications d'Invitation
 chatroom_adm#:#general_settings_title#:#Paramètres Généraux du Chat
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Intervalle de rafraîchissement (secondes)
+chatroom_adm#:#osd_interval#:#Intervalle de rafraîchissement (secondes)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.